src/PerformanceTests/NServiceBus5/App.config

```diff
@@ -1,26 +1,49 @@
-<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+﻿<?xml version="1.0" encoding="utf-8" standalone="yes"?>
 <configuration>
   <configSections>
     <section name="MessageForwardingInCaseOfFaultConfig" type="NServiceBus.Config.MessageForwardingInCaseOfFaultConfig, NServiceBus.Core" />
     <section name="AuditConfig" type="NServiceBus.Config.AuditConfig, NServiceBus.Core" />
     <section name="RijndaelEncryptionServiceConfig" type="NServiceBus.Config.RijndaelEncryptionServiceConfig, NServiceBus.Core" />
     <section name="Logging" type="NServiceBus.Config.Logging, NServiceBus.Core" />
-    <section name="nlog" type="NLog.Config.ConfigSectionHandler, NLog" />
+    <section name="log4net" type="log4net.Config.Log4NetConfigurationSectionHandler, log4net" />
   </configSections>
 
-  <nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-    <targets async="true">
-      <target name="file" xsi:type="File" fileName="trace.log" layout="${longdate:universalTime=true}|${level:uppercase=true}|${threadid:padding=2}|${logger}|${message}${onexception:${newline}${exception:format=tostring}}" />
-      <target name="trace" xsi:type="OutputDebugString" layout="${level:uppercase=true}|${threadid:padding=2}|${logger}|${message}${onexception:${newline}${exception:format=tostring}}" />
-      <target name="console" xsi:type="ColoredConsole" layout="${longdate}|${level:uppercase=true}|${threadid:padding=2}|${logger}|${message}${onexception:${exception:format=message}}" />
-    </targets>
-    <rules>
-      <logger name="NServiceBus.Azure.Transports.WindowsAzureServiceBus.AzureServiceBusQueueCreator" maxlevel="Info" final="true" />
-      <logger name="NServiceBus.Distributor.MSMQ.*" maxlevel="Info" final="true" />
-      <logger name="NServiceBus.RecoverabilityExecutor" maxlevel="Info" final="true" />
-      <logger name="*" minlevel="Debug" writeTo="file,trace,console" />
-    </rules>
-  </nlog>
+  <log4net>
+    <appender name="Console" type="log4net.Appender.ColoredConsoleAppender" >
+      <mapping>
+        <level value="ERROR" />
+        <foreColor value="Yellow" />
+      </mapping>
+      <mapping>
+        <level value="WARN" />
+        <foreColor value="Purple" />
+      </mapping>
+      <mapping>
+        <level value="FATAL" />
+        <foreColor value="RED" />
+      </mapping>
+      <layout type="log4net.Layout.PatternLayout">
+        <conversionPattern value="%date|%level|%thread|%logger|%message%newline%exception" />
```

Comment:
The console logger previously didn't log the exception to keep the information readable.

Reply:
@ramonsmits it used this before: `${onexception:${exception:format=message}}` which tbh I don't fully understand as this is nlog specific syntax, but it does look like it would output the exception message?

Is this really an issue?

Reply:
@timbussmann That line only writes the exception message, not the stack trace. This is really convenient when looking at the output during execution. It is not a must but these days with our huge stack traces you do not see anything else on screen.

Reply:
I don't see a way to emulate this behavior in log4net.

Reply:
I would want to see the exception message, but not the stack trace. I just read something about `<IgnoresException value="False" />` but I'm not sure what that does exactly.